packages/slice-machine/src/components/GenerateSliceWithAiModal/SliceCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { Card, CardFooter, CardMedia } from "@/components/Card";
+
+import { Slice } from "./GenerateSliceWithAiModal";
```

Comment:
we import something from the modal here, and then we import this card from the modal, creating a circular reference 

in this case is not a problem because one is a type, but it'd feel better if you declare the slice type within the slice card, and then import both the slice type and the slice card to build the more complex slice modal